wbooks/src/app/components/AppNavigator/index.tsx

```diff
@@ -17,4 +22,24 @@ function AppNavigator() {
   );
 }
 
+function WishListScreen() {
+  return <View />;
+}
+
+function AppNavigator() {
+  return (
+    <Tab.Navigator
+      screenOptions={({ route }) => ({
+        tabBarIcon: ({ focused }) => <TabIcon focused={focused} route={route} />
+      })}
+      tabBarOptions={{
+        activeTintColor: COLORS.blue,
+        inactiveTintColor: COLORS.gray
+      }}>
+      <Tab.Screen name={TABS.bookList} component={LibraryStackScreen} />
+      <Tab.Screen name={TABS.wishList} component={WishListScreen} />
```

Comment:
Use https://reactnavigation.org/docs/bottom-tab-navigator#options for Tab label title with `option` like in `LibraryStackScreen`